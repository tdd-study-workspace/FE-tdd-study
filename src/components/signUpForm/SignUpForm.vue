<template>
  <div>
    <input id="username" type="text" placeholder="username" />
    <input id="email" type="text" placeholder="email" />
    <input id="password" type="text" placeholder="password" />
  </div>
</template>

<script setup lang="ts">
  
</script>

<style scoped>
  
</style>