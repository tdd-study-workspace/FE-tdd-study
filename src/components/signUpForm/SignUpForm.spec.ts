import { describe, it } from "vitest";
import { render, screen } from "@testing-library/vue";
import SignUpForm from "./SignUpForm.vue";

describe("SignUpForm 테스트", () => {
  it("SignUpForm 렌더링 시 내부에 UserName, Email, Password INPUT 3개가 표시되어야함", () => {
    render(SignUpForm);
    screen.getByRole("input");
  });
});
