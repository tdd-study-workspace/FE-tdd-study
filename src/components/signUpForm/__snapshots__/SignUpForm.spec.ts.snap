// Vitest 

exports[`SignUpForm 테스트 > matches snapshot 1`] = `
<div>
  <div>
    <input
      id="username"
      placeholder="username"
      type="text"
    />
    <input
      id="email"
      placeholder="email"
      type="text"
    />
    <input
      id="password"
      placeholder="password"
      type="text"
    />
  </div>
</div>
`;
